POM/Hotels/HotelsHome.js

```diff
@@ -42,6 +44,357 @@ class HotelsHome {
+    espanolButton = '//div[contains(text(),"Español")]'
+    englishOption =  '//option[contains(text(),"English (United States)")]'
+    spanishSaveButton = "//button[contains(text(),'Guardar')]"
+    
+    calenderButton = '//button[@id="date_form_field-btn"]'
+    backMonthButton = "//div[@class='uitk-calendar']//div//button"
+    septemberMonth = '//h2[contains(text(),"September")]'
+
+    property = '//div[contains(text(),"List your property")]'
+    wouldYouLike = '//p[contains(text(),"What would you like to list?")]'
+    lodging = '//p[contains(text(),"Lodging")]'
+    privateResidence = '//p[contains(text(),"Private residence")]'
+    
+    step1 = '//div[contains(text(),"Step 1 of 3")]'
+    step2 = '//div[contains(text(),"Step 2 of 3")]'
+    step3 = '//div[contains(text(),"Step 3 of 3")]'
+
+    earn = '//h1[contains(text(),"See how much you could earn!")]'
+    
+    plusBedroom = '//button[@aria-label="Increase bedrooms"]'
+    plusBathroom= '//button[@aria-label="Increase bathrooms"]'
+
+    nextButton ='//button[@id="propertyInfoNextBtn"]'
+
+    propertyLocated = '//h1[contains(text(),"Where is your property located?")]'
+    enterAddress = '//input[@id="locationTypeAhead"]'
+    addressAuto = '//ul[@class="typeahead-prediction-list fds-field-select"]'
+
+    map = '//div[@data-wdio="google-map-component"]'
+    pinInMap = "//span[contains(text(),'To navigate')]/following-sibling::div"
+    insideTheAuto = '//ul[@class="typeahead-prediction-list fds-field-select"]'
+    pinLocationVerify = '//div[contains(text(),"Pin location may not be exact.")]'
+    
+    goingToLocator = "[aria-label='Going to']"
+    
+    selectDates = '//button[@id="date_form_field-btn"]'
+    clickNextMonth = '[aria-label="Next month"]'
+
+    clickingApplyLocator = '//button[@data-stid="apply-date-picker"]'
+    clickingSearchLocator = '//button[@id="submit_button"]'
+    
+    tellUsHow = '//span[contains(text(),"Tell")]'
+    feedBack = '//a[contains(text(),"Share feedback")]'
+
+    star5 = '//span[contains(text(),"5★")]'
+    sortBy = '//select[@id="sort"]'
+    optionPrice ='//option[@value="PRICE_LOW_TO_HIGH"]'
+    firstStarRating = '//body/div[@id="app-shopping-pwa"]/div[@id="app-layer-manager"]/div[@id="app-layer-base"]/div[1]/main[1]/div[1]/div[1]/div[1]/div[1]/div[1]/section[1]/div[1]/div[2]/div[1]/div[2]/section[2]/ol[1]/li[2]/div[1]/div[1]/div[2]/div[1]/div[1]/div[1]'
+    secondStarRating = '//body/div[@id="app-shopping-pwa"]/div[@id="app-layer-manager"]/div[@id="app-layer-base"]/div[1]/main[1]/div[1]/div[1]/div[1]/div[1]/div[1]/section[1]/div[1]/div[2]/div[1]/div[2]/section[2]/ol[1]/li[3]/div[1]/div[1]/div[2]/div[1]/div[1]/div[1]'
+    thirdStarRating ='//body/div[@id="app-shopping-pwa"]/div[@id="app-layer-manager"]/div[@id="app-layer-base"]/div[1]/main[1]/div[1]/div[1]/div[1]/div[1]/div[1]/section[1]/div[1]/div[2]/div[1]/div[2]/section[2]/ol[1]/li[4]/div[1]/div[1]/div[2]/div[1]/div[1]/div[1]'
```

Comment:
Above 3 locators are not good, got too many dependencies, may break any time. Create your own custom xpath.